source/tutorials/multi-boot/dual-boot-win.rst

```diff
@@ -237,10 +247,135 @@ If you prefer not to use your BIOS to load the :guilabel:`Boot Menu` and select
 #. You should be presented with the :command:`systemd-boot` menu, as shown
    below.
 
+   .. rst-class:: dropshadow
+
    .. figure:: ../../_figures/multi-boot/dual-boot-win-04.png
       :scale: 85%
       :alt: systemd-boot menu
 
       Figure 5: systemd-boot menu
 
 
+Alternative: Install Windows 10 OS After |CL|
+*********************************************
+
+The following alternative guide shows how to install Windows 10 OS adjacent
+to an existing |CL| installation.
+
+Prerequisites
+*************
+
+* |CL| is already installed.  
+* There is unallocated disk space available. If the entire disk has been
+  allocated to |CL|, then shrink the root partition to make space for 
+  Windows 10 OS.
+
+Install Windows 10 OS
+=====================
+
+#. Start the Windows installer and follow the prompts.
+
+#. At the Type of installation screen, choose 
+   :guilabel:`Custom: Install Windows only (advanced)`. See Figure 6.
+
+   .. rst-class:: dropshadow
+
+   .. figure:: ../../_figures/multi-boot/dual-boot-win-06.png
+      :scale: 85%
+      :alt: Windows - Choose installation type
+
+      Figure 6: Windows - Choose installation type
+
+#. Select :guilabel:`Unallocated Space` and create a new partition of 
+   the desired size. For this example, we will use the entire unallocated 
+   space. See Figure 7.
+
+   .. rst-class:: dropshadow
+
+   .. figure:: ../../_figures/multi-boot/dual-boot-win-07.png
+      :scale: 85%
+      :alt: Windows - Create new partition
+
+      Figure 7: Windows - Create new partition
+
+  .. note::
+     
+     Normally, Windows creates its own 100 MB EFI partition if none exists. 
+     In our case, where an EFI partition was created by |CL|, Windows will 
+     use the previously-created partition.
+
+#. Select the newly-created partition and follow the remaining prompts to 
+   complete the Windows installation. See Figure 8.
+
+   .. rst-class:: dropshadow
+
+   .. figure:: ../../_figures/multi-boot/dual-boot-win-08.png
+      :scale: 85%
+      :alt: Windows - Install on newly-created partition
+
+      Figure 7: Windows - Install on newly-created partition
+
+#. Finish the Windows out-of-box-experience process.
+
+#. Restore `systemd-boot` - the bootloader |CL| uses - and add Windows to 
+   its boot menu.
+
+   At this point, you cannot boot |CL| because Windows became the default 
+   bootloader after it was installed.  Follow these steps to restore 
+   `systemd-boot`.
+
+   a. Boot up the |CL| installer. 
```

Comment:
Make this a hyperlink, please (some users may not be familiar with our docs): 

Boot up the |CL|  :ref: `installer image<install-on-target-start>`
Note: Start tickmark after colon in :ref: . Add final tickmark after `>`

Please make the exact same change on Ln 153, above under Advanced systemd option too.